src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2150,6 +2150,8 @@ uint32_t RadioSpinel<InterfaceType, ProcessContextType>::GetBusSpeed(void) const
 template <typename InterfaceType, typename ProcessContextType>
 void RadioSpinel<InterfaceType, ProcessContextType>::HandleRcpUnexpectedReset(spinel_status_t aStatus)
 {
+    OT_UNUSED_VARIABLE(aStatus);
```

Comment:
What if the log level is set to `NONE`?

Reply:
I think this change is required when log level is `NONE`.